Project.toml

```diff
@@ -31,6 +31,7 @@ julia = "1.6"
 BlackBoxOptim = "a134a8b2-14d6-55f6-9291-3336d3ab0209"
 DelayDiffEq = "bcd4f6db-9728-5f36-b5f7-82caef46ccdb"
 ForwardDiff = "f6369f11-7733-5829-9624-2563aa707210"
+LeastSquaresOptim = "0fc2ff8b-aaa3-5acd-a817-1944a5e08891"
```

Comment:
Make it an extension package?

Reply:
Even this is not needed  - LeastSquaresOptim is only in the weakdeps for testing.